packages/trader/src/Modules/MT5/Components/mt5-account-card.jsx

```diff
@@ -100,12 +104,17 @@ const MT5AccountCard = ({
                             {Object.keys(specs).map((spec_attribute, idx) => (
                                 <tr key={idx} className='mt5-account-card__specs-table-row'>
                                     <td className='mt5-account-card__specs-table-attribute'>
-                                        <p className='mt5-account-card--paragraph'>{spec_attribute}</p>
+                                        <Text as='p' className='mt5-account-card--paragraph'>
```

Comment:
Can we pass the props like size, weight and etc to the Text component instead of passing the className? It seems that needed props are available in the text component.